pages/design/rule-design.md

```diff
@@ -39,34 +77,64 @@ A test aspect is a part of a test subject that must be available in order to pro
 
 Other aspects may be necessary for testing. These can be added as long as they are sufficiently defined.
 
+For more details, see [ACT Rules Format: Input](https://www.w3.org/TR/act-rules-format/#input).
+
 ### Authors
 
-Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`.
+Names of the Authors. These must be an exact match of names in `contributors` property in `package.json`. Note: If an author does not exist in `package.json`, they should be added to that file as part of the relevant PR.
+
+## Applicability
```

Comment:
Currently, out of 53 rules we seem to have 30 applicabilities that start with "The rule applies to", 11 start with "This rule applies to" and 12 in other way (including an ugly "The rule only applies to").

=> I think we should make a choice on how to unify this (and then "The rule applies to" seem to be the majority case, my personal preference would rater be "This rule applies to") and document this choice here.

Reply:
I also prefer "This rule ..." and "This example ..." for these declarative phrases.

Reply:
Marking this as resolved, as no-one else has commented.